spyder/plugins/completion/kite/widgets/calltoaction.py

```diff
@@ -0,0 +1,151 @@
+                            QPushButton)
+from spyder.utils.syntaxhighlighters import get_color_scheme
+
+from spyder.config.base import _
+from spyder.config.gui import get_font
+from spyder.config.manager import CONF
+from spyder.plugins.completion.kite.bloomfilter import KiteBloomFilter
+from spyder.plugins.completion.kite.parsing import find_returning_function_path
+from spyder.plugins.completion.kite.utils.status import check_if_kite_installed
+from spyder.plugins.completion.fallback.actor import FALLBACK_COMPLETION
+
+COVERAGE_MESSAGE = (
+    _("No completions found."
+      " Get completions for this case and more by installing Kite.")
+)
+
+
+class KiteCallToAction(QFrame):
+    def __init__(self, textedit, ancestor):
+        super(KiteCallToAction, self).__init__(ancestor)
+        self.textedit = textedit
+
+        self.setObjectName("kite-call-to-action")
+        self.set_color_scheme(CONF.get('appearance', 'selected'))
+        # Reuse completion window size
+        # self.setFont(get_font())
+        self.setFocusPolicy(Qt.NoFocus)
+
+        # sub-layout: horizontally aligned links
+        actions = QFrame(self)
+        actions_layout = QHBoxLayout()
+        actions_layout.setContentsMargins(5, 5, 5, 5)
+        actions_layout.setSpacing(10)
+        actions_layout.addStretch()
+        actions.setLayout(actions_layout)
+
+        self._install_button = QPushButton(_("Install Kite"))
+        self._learn_button = QPushButton(_("Learn More"))
+        self._dismiss_button = QPushButton(_("Dismiss Forever"))
+        self._install_button.clicked.connect(self._install_kite)
+        self._learn_button.clicked.connect(self._learn_more)
+        self._dismiss_button.clicked.connect(self._dismiss_forever)
+        actions_layout.addWidget(self._install_button)
+        actions_layout.addWidget(self._learn_button)
+        actions_layout.addWidget(self._dismiss_button)
+
+        # main layout: message + horizontally aligned links
+        main_layout = QVBoxLayout()
+        main_layout.setContentsMargins(5, 5, 5, 5)
+        self.setLayout(main_layout)
+        self.label = QLabel(self)
+        self.label.setWordWrap(True)
+        main_layout.addWidget(self.label)
+        main_layout.addWidget(actions)
+        main_layout.addStretch()
+
+        self._enabled = CONF.get('kite', 'call_to_action')
+        self._escaped = False
+        self.hide()
+
+        is_kite_installed, __ = check_if_kite_installed()
+        if is_kite_installed:
+            self._dismiss_forever()
+
+    def set_color_scheme(self, color_scheme):
+        if not isinstance(color_scheme, dict):
+            color_scheme = get_color_scheme(color_scheme)
+        bg_color = color_scheme['background']
+        border_color = color_scheme['sideareas']
+        text_color, __, __ = color_scheme['normal']
+        button_color = color_scheme['currentline']
+        hover_color = color_scheme['currentcell']
```

Comment:
I think using our color scheme for the CTA colors is the wrong thing to do. I mean `color_scheme['normal']` is fine, but the other colors vary wildly aming color schemes. For instance, using `color_scheme['background']` in light themes makes no sense because that color is `white`. And `color_scheme['currentline']` is a shade purple in other schemes.

@metalogical, I think you should use the same colors as the completion widget for background and foreground, and also preserve the same colors and hover behavior for buttons as any other widget.

Reply:
done: I had to use `setFrameStyle` to force the frame style to match `QListWidget`, use `setAutoFillBackground` to prevent the frame from being transparent in the Light UI mode, and copied the window flags from the completion widget.

It still doesn't look 100% identical, but extremely close: the completions widget has rounded corners, and in light mode, has a blue highlight around the border. Not sure how that happens.